lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -53,7 +53,8 @@ module CreditCardMethods
         'hipercard' => ->(num) { num&.size == 16 && in_bin_range?(num.slice(0, 6), HIPERCARD_RANGES) },
         'panal' => ->(num) { num&.size == 16 && in_bin_range?(num.slice(0, 6), PANAL_RANGES) },
         'verve' => ->(num) { (16..19).cover?(num&.size) && in_bin_range?(num.slice(0, 6), VERVE_RANGES) },
-        'tuya' => ->(num) { num =~ /^588800\d{10}$/ }
+        'tuya' => ->(num) { num =~ /^588800\d{10}$/ },
+        'uatp' => ->(num) { num =~ /^1\d{14}$/ }
```

Comment:
This is going to validate every card that starts with a 1, but the criteria for this card is that it starts with a `1` AND is followed by a a 3-digit IIN. Since the IIN we were given is `175`, only cards that start `1175` should validate.